src/pages/integrations/bitpay-card/bitpay-card-topup/bitpay-card-topup.ts

```diff
@@ -646,9 +699,18 @@ export class BitPayCardTopUpPage {
     this.remainingTimeStr = ('0' + m).slice(-2) + ':' + ('0' + s).slice(-2);
   }
 
+  logLegacyCardSetCheckoutOption(wallet) {
+    this.bitPayCardProvider.logEvent('set_checkout_option', {
```

Comment:
This log shouldn't be done once the payment is made?
Is it okay to send a new log every time you change the selected wallet?